Common/Models/SetBolusUserInfo.swift

```diff
@@ -15,6 +15,7 @@ struct SetBolusUserInfo {
     let startDate: Date
     let contextDate: Date?
     let carbEntry: NewCarbEntry?
+    let activationType: BolusActivationType
```

Comment:
This type is appropriate for Loop, maybe less so for the watch, as there is no automatic bolus possible from there. I won't block for this, but it might be a little cleaner to have a type that fits this use case more specifically, and just has three states; noRecommendation, acceptedRecommendation and changedRecommendation. Loop can map this when it needs to to the appropriate Loop type. Another type, but less entanglement, and more separation of concerns.